spec/lib/wysiwyg_sanitizer_spec.rb

```diff
@@ -1,8 +1,8 @@
 require 'rails_helper'
 
-describe WYSIWYGSanitizer do
+describe 'WYSIWYGSanitizer' do
```

Comment:
It looks like the changes using strings introduced in 424dfdf were reverted later. To keep it consistent, what do you think about changing the first line back to `describe WYSIWYGSanitizer do`?